src/swapRouter.ts

```diff
@@ -469,6 +467,22 @@ export abstract class SwapRouter {
     }
   }
 
+  // if slippage is very high, there's a chance of hitting max/min prices resulting in a partial fill of the swap
+  private static riskOfPartialFill(trades: AnyTradeType): boolean {
+    const FIFTY_PERCENT = new Percent(JSBI.BigInt(50), JSBI.BigInt(100))
```

Comment:
Nit: constant at top of file, and named more descriptively like: `REFUND_ETH_PRICE_IMPACT_THRESHOLD`